config/config/common-config.yaml

```diff
@@ -660,7 +660,7 @@ networkPolicies:
         - 443
 
   rookCeph:
-    enabled: true
+    enabled: false
```

Comment:
Would we like to add to like the init command to set this to true if it's a exoscale cluster?

Reply:
seems that with rook network policies enabled we cannot create PVCs
you can see that [here](https://github.com/elastisys/compliantkubernetes-apps/actions/runs/3702298246/jobs/6272466621)

I will add the init script update in the task opened to fix the rook network policies.

Reply:
@crssnd Hmmm interesting , because when i tried to create manually , it worked :( !!